M/MPFI/bundled/patches/remove-doc-build.patch

```diff
@@ -0,0 +1,9 @@
+--- Makefile.am.original    2020-05-01 20:07:47.404699308 +0100
++++ Makefile.am 2020-05-01 20:07:51.244721260 +0100
+@@ -1,5 +1,5 @@
+ ACLOCAL_AMFLAGS = -I m4
+-SUBDIRS = src tests doc
++SUBDIRS = src tests
```

Comment:
I think you can just call
```sh
make -j${nproc} SUBDIRS="src"
```
without patching anything?  BTW, no need to build the tests either, we can't run them anyway